### PR TITLE
example update to run on latest ios simulator

### PR DESCRIPTION
**Pull request by vasilich6107**
Hi! Thanks for your work.
I would like to practice a little in form usage.

I came from `flutter_form_bloc`. This project is not supported anymore. At least there is no activity from author.
So I'd like to implement examples like this https://giancarlocode.github.io/form_bloc/#/

If you are ok with this approach I'll try to improve examples)

There is my first PR aimed to update the `example` to run on latest ios simulator

### CODE REVIEW

File: example/android/build.gradle [hunk 1]
```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.3.71'
+    ext.kotlin_version = '1.3.50'
```
Comment (joanpablo) [inline example/android/build.gradle:2]: Why to downgrade versions?

File: example/android/build.gradle [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
     repositories {
         google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
+        classpath 'com.android.tools.build:gradle:3.5.0'
```
Comment (joanpablo) [inline example/android/build.gradle:9]: Why to downgrade versions?

File: example/android/settings.gradle [hunk 3]
```diff
@@ -1,15 +1,11 @@
...
+localPropertiesFile.withReader("UTF-8") { reader -> properties.load(reader) }
 
-plugins.each { name, path ->
-    def pluginDirectory = flutterProjectRoot.resolve(path).resolve('android').toFile()
-    include ":$name"
-    project(":$name").projectDir = pluginDirectory
-}
+def flutterSdkPath = properties.getProperty("flutter.sdk")
+assert flutterSdkPath != null, "flutter.sdk not set in local.properties"
+apply from: "$flutterSdkPath/packages/flutter_tools/gradle/app_plugin_loader.gradle"
```
Comment (joanpablo) [inline example/android/settings.gradle:11]: I'm not really sure about this changes.

the load of **local properties** and **flutter sdk** are already included in "example/android/app/**build.gradle**".

and I'm a little confuse about why you have deleted code that was generated by Flutter by default.

like:
```dart
def flutterProjectRoot = rootProject.projectDir.parentFile.toPath()

def plugins = new Properties()
def pluginsFile = new File(flutterProjectRoot.toFile(), '.flutter-plugins')
if (pluginsFile.exists()) {
    pluginsFile.withReader('UTF-8') { reader -> plugins.load(reader) }
}

plugins.each { name, path ->
    def pluginDirectory = flutterProjectRoot.resolve(path).resolve('android').toFile()
    include ":$name"
    project(":$name").projectDir = pluginDirectory
}
```

**Review by joanpablo — CHANGES_REQUESTED** [review]
That's really great if you could improve examples, thanks a lot for that contribution.

I've just left some comments about changes that are not really clear to my before merge this pull request.

I'm very glad to have you as a contributor.

**Comment by vasilich6107** [comment]
Sorry I forgot to mention that this PR does not actually require reviewing)
I will explain what I mean.

Previously I had bad experience while trying to make my project compatible with latest xCode.
So I did not make an attempt to fix the compatibility issues manually.

As far as `example` does not affect the native side I just removed whole example project and created new one with `flutter create` command. Then reverted back the contents of `pubspec`, `readme` and `lib` folder. Easy fix with almost no stress ;-)

So all you see here is the code that shipped with the latest flutter `1.22.2`

**Review by joanpablo — APPROVED** [review]
I'm in flutter 1.22.0 so I'm OK with 1.22.2.

Thanks again to contribute with Reactive Forms.

Welcome aboard!!!

**Pull request merged**